README.md

```diff
@@ -20,7 +21,7 @@ Add a report portal appender into `logback.xml` configuration file.
 </root>
 ```
 
-#### Screenshots
+#### Attaching files 
```

Comment:
`Attaching file (attaching screenshots, etc.)`

people will use google to find it. that kind of naming will be more helpful.

Reply:
done